### PR TITLE
[FTS-1501] Implement interest and overdue

### PR DESCRIPTION
**Pull request by shivam21**

### CODE REVIEW

**Review by lovemhjn — APPROVED** [review]

File: src/main/java/lib/dehaat/ledger/presentation/ledger/LedgerDetailActivity.kt [hunk 1]
```diff
@@ -10,155 +10,184 @@ import androidx.activity.compose.setContent
...
+		}
+
+		if (!LedgerSDK.isCurrentAppAvailable()) {
+			showToast(getString(R.string.initialise_ledger))
+			finish()
+			return
+		}
+
+		if (args.partnerId.isEmpty() && LedgerSDK.isDebug) {
+			showToast("PartnerId missing (check dehaat-center-API)")
```
Comment (garghimanshu0786) [inline src/main/java/lib/dehaat/ledger/presentation/ledger/LedgerDetailActivity.kt:67]: add string resource

Comment (shivam21) [reply src/main/java/lib/dehaat/ledger/presentation/ledger/LedgerDetailActivity.kt:67]: this is for debug only

File: src/main/java/lib/dehaat/ledger/presentation/ledger/invoicelist/WidgetInvoiceListVM.kt [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	val uiState = viewModelState.map { it.toUiState() }.stateIn(
+		viewModelScope, SharingStarted.Eagerly, viewModelState.value.toUiState()
+	)
+
+	init {
+		getInvoicesList()
+		initBottomBarData()
+	}
+
+	private fun initBottomBarData() {
```
Comment (garghimanshu0786) [inline src/main/java/lib/dehaat/ledger/presentation/ledger/invoicelist/WidgetInvoiceListVM.kt:64]: remove this method

File: src/main/java/lib/dehaat/ledger/presentation/ledger/ui/component/creditSummary/BlockOrderingWidget.kt [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+			modifier = Modifier
+				.padding(start = 6.dp)
+				.fillMaxHeight(),
+			contentScale = ContentScale.FillHeight
+		)
+	}
+
+@Preview
+@Composable
+fun ColumnScope.LineWithCircle(color: Color = Color.Black) {
```
Comment (garghimanshu0786) [inline src/main/java/lib/dehaat/ledger/presentation/ledger/ui/component/creditSummary/BlockOrderingWidget.kt:87]: Add in common composables

Comment (shivam21) [reply src/main/java/lib/dehaat/ledger/presentation/ledger/ui/component/creditSummary/BlockOrderingWidget.kt]: This is not so common

File: src/main/java/lib/dehaat/ledger/presentation/ledger/invoicelist/WidgetInvoiceListVM.kt (hunk 2 above)
Comment (shivam21) [reply src/main/java/lib/dehaat/ledger/presentation/ledger/invoicelist/WidgetInvoiceListVM.kt:64]: done

**Review by garghimanshu0786 — APPROVED** [review]

**Comment by shivam21** [comment]
merged in other PR

**Pull request closed without merging**